tests/conftest.py

```diff
@@ -428,10 +428,12 @@ def investigate_timeout(self, reason):
         """if some kind of client request observes a timeout, investigate the
         herbstluftwm server process. 'reason' is best phrased using present
         participle"""
+        self.read_and_echo_output()
```

Comment:
I've added this during debugging, but this should stay (this is important because before, error messages didn't appear in the travis output if a hc-command made herbstluftwm quit). Do you want me to move it to a separate PR?

Reply:
I see! Alright, just keep it in this change